ModelCompiler/README.md

```diff
@@ -0,0 +1,49 @@
+# Generating E2e Model to IREE
+
+This IREE Samples subdirectory is meant for e2e model compiling from Tensorflow to either generating MLIR that can be compiled through iree translate (i.e std,linalg,iree,util dialect land) or to be compiled to run straight away.
+
+We have three different types of python scripts:
+
+- python scripts that are meant to generate MLIR will be prefixed by \<model\_name\>\_**gen**.py 
+- python scripts that are meant to compile and RUN will be prefixed by \<model\_name\>\_**run**.py
+- python scripts that are meant to RUN with regular tf will be prefixed by \<model\_name\>\_**tf**.py
+
+## Initial/First Time Setups
+### VirtualEnv Setup
+```bash
+cd ~
+pip install virtualenv
+virtualenv -p /usr/bin/python3.9 ModelCompilerEnv
+source ~/ModelCompilerEnv/bin/activate
+```
+
+### Installing Miscallaneous Deps
+```bash
+pip install tensorflow
```

Comment:
It'd be great to just have a requirements.txt file. See this one for an example that also uses the find-links method: https://github.com/google/iree/blob/main/docs/api_docs/python/requirements.txt

Reply:
> It'd be great to just have a requirements.txt file. See this one for an example that also uses the find-links method: https://github.com/google/iree/blob/main/docs/api_docs/python/requirements.txt

> I'm open to trying it either way (hub or submodule) and seeing what works. I have a slight preference for hub when possible because it has the least systemic impact on folks using this repo.

@hcindyl Ooo, sorry, I thought you meant like "download" = manual copy source code to this directory. Now I realize tf hub is a python module hahaha.
I am actually not super familiar with TF Hub, I'll see if we can change it to that, since I also agree that we shouldn't add new submodules when possible :)

SG! will change it to requirements.txt